src/storage/index/LookupProcessor.cpp

```diff
@@ -11,7 +11,7 @@ namespace storage {
 
 void LookupProcessor::process(const cpp2::LookupIndexRequest& req) {
     if (executor_ != nullptr) {
-        executor_->add([req, this] () {
+        executor_->add([this, req = std::move(req)] () mutable {
```

Comment:
Move the constant variable is useless.

Reply:
> Move the constant variable is useless.

<img width="736" alt="image" src="https://user-images.githubusercontent.com/56461666/105455957-5df86180-5cbf-11eb-8a88-42ab2581064c.png">
I believe that in all things there is a purpose, not ready to review. hold this PR.